docs/conf.py

```diff
@@ -516,3 +516,8 @@ def setup(app):
         app.connect("include-read", rstjinja_include)
     except BaseException as e:
         print("cannot do jinja-replacements on included files")
+
+
+def setup(app):
+    app.add_js_file("sticky_video.js")
```

Comment:
there is already an `setup` method, so I don't think this will do anything. At least I don't think it should be necessary.